pkg/config/config.go

```diff
@@ -18,3 +18,47 @@ type Config struct {
 	Metadata *Metadata `json:"metadata" yaml:"metadata"`
 	Rollout  *Rollout  `json:"rollout" yaml:"rollout"`
 }
+
+// WithValues initializes a configuration with the given values.
+func WithValues(project, region, service string, steps []int64, interval int64) *Config {
+	return &Config{
+		Metadata: &Metadata{
```

Comment:
do not call it metadata. think of extensibility in the future.

```
rollouts:
- stategy: {steps: ..., interval : .... }
  targets:
  - service:
      name: x
      project: y
  - service:
      name: x
      project: y

- stategy: {steps: ..., interval : .... }
  targets:
  - serviceSelector:
      foo: bar
```

Reply:
I will do this in the next PR